va-gov/layouts/home.html

```diff
@@ -164,17 +164,17 @@ <h4><a href=/housing-assistance/><i class="icon-small fa fa-home"></i>Housing As
       <div class="usa-grid">
 
         <div class="usa-width-one-third">
-          <h4><a href=/life-insurance/><i class="icon-small fa fa-users"></i>Life Insurance</a></h4>
+          <h4><a href="/life-insurance/" onclick="recordEvent({ event: 'nav-linkslist' });"><i class="icon-small fa fa-users"></i>Life Insurance</a></h4>
           <p>Explore VA life insurance options for Veterans, Servicemembers, and families. Manage your policy online, file claims for benefits, and access helpful resources.</p>
         </div>
 
         <div class="usa-width-one-third">
-          <h4><a href=/burials-memorials/><i class="icon-small fa fa-star"></i>Burials and Memorials</a></h4>
+          <h4><a href="/burials-memorials/" onclick="recordEvent({ event: 'nav-linkslist' });"><i class="icon-small fa fa-star"></i>Burials and Memorials</a></h4>
           <p>Get help planning a burial in a VA national cemetery, order a headstone or other memorial item to honor a Veteran's service, and apply for survivor and dependent benefits.</p>
         </div>
 
         <div class="usa-width-one-third">
-          <h4><a href=/records/><i class="icon-small fa fa-id-card"></i>Records</a></h4>
+          <h4><a href="/records/" onclick="recordEvent({ event: 'nav-linkslist' });"><i class="icon-small fa fa-id-card"></i>Records</a></h4>
           <p>Apply for a printed Veteran ID card, get your VA benefit letters and medical records, and learn how to apply for a discharge upgrade.</p>
```

Comment:
I considered querying for these elements and attaching event listeners, but this explicitness seemed more consistent with out style for collecting analytics throughout Metalsmith files.